web/src/components/itemList.js

```diff
@@ -89,8 +94,8 @@ class ItemList extends Component {
                            const selected = [];
                            const currency = [];
                            for(let i = 0; i < data.getItems.length; i++) {
-                               selected.push(false);
-                               currency.push(false);
+                               selected.push(0);
+                               currency.push(0);
```

Comment:
Why did you change the type of items that are inserted into currency from boolean to integer?